man/datatable-condition-classes.Rd

```diff
@@ -0,0 +1,38 @@
+\name{data.table-error-handling}
+\alias{data.table-error-handling}
+\title{Condition Handling with Classed Conditions}
+\description{
+\code{data.table} provides specific condition classes for common operations, making it easier to handle comditions programmatically. This is particularly useful when writing robust code or packages that use \code{data.table}. Relying on the exact text of condition messages is fragile (it is not uncommon to change the wording slightly, or for the user's session not to be in English); prefer using the signal class where possible.
+}
+\details{
+\subsection{Available error Classes}{
```

Comment:
here we still refer only to "error"

Reply:
SGTM, however, we'll need a section for warnings already:

https://github.com/Rdatatable/data.table/blob/a0808332b556654e1493db44abedc7fc22c197a1/R/fcast.R#L178

Reply:
Shouldn't the GHA be failing since we missed this `dt_missing_fun_aggregate_warning`?

Reply:
Yes, see my commits -- some typos/"error" references missed